src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -159,28 +166,25 @@ const PublicationForm = ({
                 <p>or</p>
 
                 <div className={styles.optionsContainer}>
-                  <FormRadioGroup
+                  <RHFFormFieldRadioGroup
                     id={`${formId}-themes`}
```

Comment:
We should be able to get rid of this id as we can just use the auto-generated one